kvmagent/kvmagent/plugins/virtual_pci_device/virtualization.py

```
@@ -0,0 +1,20 @@
+from abc import ABCMeta, abstractmethod
+
+
+class VirtualizationHandler(object):
+    __metaclass__ = ABCMeta
+    @abstractmethod
+    def get_all_devices(self):
+        pass
+
+    @abstractmethod
+    def get_all_devices(self):
```

Ruff (0.14.5)

Redefinition of unused `get_all_devices` from line 7

(F811)